src/deepsparse/pipelines/function_pipeline.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Callable, Dict, List, Tuple, Type, Union
+
+import numpy
+from pydantic import BaseModel
+
+from deepsparse.pipeline import Pipeline
+from deepsparse.utils.onnx import model_to_path
+
+
+@Pipeline.register(task="custom")
+class FunctionPipeline(Pipeline):
```

Comment:
I'd prefer to rename this back to CustomPipeline as originally required -- this is how we'll be referring to it in docs and custom is a much more natural thing to search for than function

Reply:
Thoughts on adding some more wording to CustomPipeline, maybe `CustomProcessingPipeline` or `CustomFunctionPipeline` or something else? It doesn't feel specific enough about what was custom about it or when a user should use this. Maybe `UnsupportedTaskPipeline` to highlight that it should be used when the task a user wants to run isn't implemented? what use case is this covering for a user?

Reply:
Maybe `CustomTaskPipeline`?

Reply:
I renamed to CustomPipeline, I am still in favor of CustomTaskPipeline though 😄

Reply:
I like CustomTaskPipeline :)